docs/datasets/danish.md

```diff
@@ -829,6 +829,72 @@ $ euroeval --model <model-id> --dataset hellaswag-da
 ```
 
 
+### GoldenSwag-da
```

Comment:
Missing the "Unofficial: " prefix here, and also for the other languages (except Portuguese).